src/js/core/methods/StellarSignTransaction.js

```diff
@@ -14,6 +15,11 @@ type Params = {
     transaction: any,
 };
 
+const StellarSignTransactionFeatures = Object.freeze({
+    manageBuyOffer: ['0', '2.4.3'],
```

Comment:
Does the first value in the list refer to the version number of Model One firmware? If it is, I think it should be set to 1.10.4

Reply:
thanks. fixed.

Reply:
Another question is, do we only need to check two operations? What about operations such as StellarCreatePassiveSellOfferOp?

I provide a complete list here.
https://github.com/trezor/connect/pull/954/files

Reply:
Feel free to investigate and add further improvements. If the other operations are not checked - user will get a less descriptive error generated by trezor-firmware. What we are doing here now is more "nice to have" than something really neccessary. Pull requests are always welcome :wink: